config/config.go

```diff
@@ -16,686 +16,634 @@ package config
+	RepositoryName                  string                    `json:"repository_name"`
+	PrintOutputType                 string                    `json:"print_output_type"`
+	JSONOutputFilePath              string                    `json:"json_output_file_path"`
+	ProjectPath                     string                    `json:"project_path"`
+	CustomRulesPath                 string                    `json:"custom_rules_path"`
+	ContainerBindProjectPath        string                    `json:"container_bind_project_path"`
+	TimeoutInSecondsRequest         int64                     `json:"timeout_in_seconds_request"`
+	TimeoutInSecondsAnalysis        int64                     `json:"timeout_in_seconds_analysis"`
+	MonitorRetryInSeconds           int64                     `json:"monitor_retry_in_seconds"`
+	ReturnErrorIfFoundVulnerability bool                      `json:"return_error_if_found_vulnerability"`
+	EnableGitHistoryAnalysis        bool                      `json:"enable_git_history_analysis"`
+	CertInsecureSkipVerify          bool                      `json:"cert_insecure_skip_verify"`
+	EnableCommitAuthor              bool                      `json:"enable_commit_author"`
+	DisableDocker                   bool                      `json:"disable_docker"`
+	EnableInformationSeverity       bool                      `json:"enable_information_severity"`
+	EnableOwaspDependencyCheck      bool                      `json:"enable_owasp_dependency_check"`
+	EnableShellCheck                bool                      `json:"enable_shell_check"`
+	SeveritiesToIgnore              []string                  `json:"severities_to_ignore"`
+	FilesOrPathsToIgnore            []string                  `json:"files_or_paths_to_ignore"`
+	FalsePositiveHashes             []string                  `json:"false_positive_hashes"`
+	RiskAcceptHashes                []string                  `json:"risk_accept_hashes"`
+	ShowVulnerabilitiesTypes        []string                  `json:"show_vulnerabilities_types"`
+	ToolsConfig                     toolsconfig.MapToolConfig `json:"tools_config"`
+	Headers                         map[string]string         `json:"headers"`
+	WorkDir                         *workdir.WorkDir          `json:"work_dir"`
+	CustomImages                    customimages.CustomImages `json:"custom_images"`
+}
+
+type Config struct {
+	GlobalOptions
+	StartOptions
+	Version string `json:"version"`
+}
+
+// New creates a new default config
+//
+// nolint:funlen
+func New() *Config {
+	wd, err := os.Getwd()
+	if err != nil {
+		logger.LogWarn("Error to get current working directory: %v", err)
+	}
 
-	return c
+	return &Config{
+		Version: version,
+		GlobalOptions: GlobalOptions{
+			ConfigFilePath: filepath.Join(wd, "horusec-config.json"),
+			LogLevel:       logrus.InfoLevel.String(),
+			LogFilePath: filepath.Join(
+				os.TempDir(), fmt.Sprintf("horusec-log-%s.log", time.Now().Format("2006-01-02 15:04:05")),
```

Comment:
NOTE: I changed the log path to be created on temp directory instead current working directory. I think that is better create log files on temp directory instead messy the directory of use with a lot of log files.